sdk/mixedreality/azure-mixedreality-remoterendering/pom.xml

```diff
@@ -0,0 +1,105 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-data-sdk-parent</artifactId>
+    <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-data-sdk-parent;current} -->
+    <relativePath>../../parents/azure-data-sdk-parent/pom.xml</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-mixedreality-remoterendering</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-mixedreality-remoterendering;current} -->
+
+  <name>Microsoft Azure SDK for Remote Rendering</name>
+  <description>This package contains Microsoft Azure SDK for Remote Rendering.</description>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.13.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-mixedreality-authentication</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-mixedreality-authentication;dependency} -->
```

Comment:
Since `azure-mixedreality-authentication` is in the same group (directory), the tag should be `current`.
```suggestion
      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-mixedreality-authentication;current} -->
```